src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1424,7 +1415,7 @@ private async Task<TokenValidationResult> ValidateTokenPayloadAsync(JsonWebToken
                 // and (since issuer validation occurs first) came from a trusted authority.
                 // NOTE: More than one nested actor token should not be considered a valid token, but if we somehow encounter one,
                 // this code will still work properly.
-                TokenValidationResult tokenValidationResult = await ValidateTokenAsync(jsonWebToken.Actor, validationParameters.ActorValidationParameters ?? validationParameters).ConfigureAwait(false);
+                TokenValidationResult tokenValidationResult = ValidateToken(jsonWebToken.Actor, validationParameters.ActorValidationParameters ?? validationParameters);
                 if (!tokenValidationResult.IsValid)
                     return tokenValidationResult;
```

Comment:
Only the async/await changes have been reverted.
Brent's change to return on actor token validation failure from [this PR](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/9ccf9536413025e189ef443ffe02e72397ab93a9) remain.

Reply:
👍